C/common/logger.cpp

```diff
@@ -29,6 +33,20 @@ inline long getCurrTimeUsec()
 	return m_timestamp.tv_usec;
 }
 
+/**
+ * Compare two strings in a case insensitive manner
+ *
+ * @param str1		The first string
+ * @param str2		The second string
+ * @return bool		True if the strings are equal, false otherwise
+ */
+bool caseInsensitiveCompare(const std::string& str1, const std::string& str2) {
```

Comment:
You don't use this, why is it here? It is not within the Logger class or belong with the Logger functionality.